src/netex_france/exporter.rs

```diff
@@ -175,7 +175,7 @@ impl<'a> Exporter<'a> {
         let from = "+proj=longlat +datum=WGS84 +no_defs"; // https://epsg.io/4326
         let to = "+proj=lcc +lat_1=49 +lat_2=44 +lat_0=46.5 +lon_0=3 +x_0=700000 +y_0=6600000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs"; // https://epsg.io/2154
         Proj::new_known_crs(from, to, None)
-            .ok_or_else(|| anyhow!("Proj cannot build a converter from '{}' to '{}'", from, to))
+            .map_err(|_| anyhow!("Proj cannot build a converter from '{}' to '{}'", from, to))
```

Comment:
I'm wondering if we should reuse original error too? The [`anyhow!()` macro](https://docs.rs/anyhow/latest/anyhow/macro.anyhow.html) can take an `Error` as argument too. But it doesn't seem possible to customize the message and keep the source error with the macro... But I'm thinking maybe `.with_context()` should be used instead?

Reply:
### test with

```rust
let from = "BAD"; // https://epsg.io/4326
let to = "+proj=lcc +lat_1=49 +lat_2=44 +lat_0=46.5 +lon_0=3 +x_0=700000 +y_0=6600000 
```

### 1. Custom message

```rust
Proj::new_known_crs(from, to, None)
    .map_err(|_| anyhow!("Proj cannot build a converter from '{}' to '{}'", from, to))
```

will display

```
Proj cannot build a converter from 'BAD' to '+proj=lcc +lat_1=49 +lat_2=44 +lat_0=46.5 +lon_0=3 +x_0=700000 +y_0=6600000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs'
```

### 2. Original Error

```rust
Proj::new_known_crs(from, to, None).map_err(|err| anyhow!(err))
```

will display

```
The underlying PROJ call failed: Unknown error (code 4096)
```

### 3. with context

```rust
use anyhow::Context;
Proj::new_known_crs(from, to, None)
            .with_context(|| format!("Proj cannot build a converter from '{}' to '{}'", from, to))
```

will display 

```
Proj cannot build a converter from 'BAD' to '+proj=lcc +lat_1=49 +lat_2=44 +lat_0=46.5 +lon_0=3 +x_0=700000 +y_0=6600000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs'

Caused by:
    The underlying PROJ call failed: Unknown error (code 4096)', 
```

We know that it will never fail and the error message will never be displayed because `from` and `to` are known and never passed as arguments.

for me

1. is ok but the message is not necessary because never displayed
2. is ok, the message says nothing but we don't care
3. 👎🏻 context does not help

Reply:
Thanks for testing and indeed, the underlying message is useless. So fine with solution 1 as you did.